### ISSUE TITLE
Datetime formats

### ISSUE THREAD

**Issue by SlimeDog** [Rsl1122/Plan-PlayerAnalytics#334]
PLAN 4.0.1

It would be nice if datetimes were formatted according to the time formats in config.yml, and aligned right when appropriate. Examples include:
- Information:Averages:SessionLength
- Information:Total:Playtime
- Sessions:50 Most Recent...:Length
- Playerlist:Playtime

**Comment by Rsl1122** [comment]
What do you mean?

**Comment by SlimeDog** [comment]
Current output:
```

Player  | Started       | Length     | World - Time
------- | ------------- | ---------- | ------------
rambeau | Oct 04, 08:42 | 2h 4m 28s  | world_blackdog (99.87%)
rambeau | Oct 04, 08:00 | 3m 31s     | world_blackdog (100%)
rambeau | Oct 03, 16:41 | 27m 52s    | world_blackdog (100%)
rambeau | Oct 03, 13:05 | 3h 17m 28s | world_yellowdog (93.71%)
```
I would like:
```
Player  | Started       | Length   | World - Time
------- | ------------- | -------- | ------------
rambeau | Oct 04, 08:42 |  2:04:28 | world_blackdog (99.87%)
rambeau | Oct 04, 08:00 |    03:31 | world_blackdog (100%)
rambeau | Oct 03, 16:41 |    27:52 | world_blackdog (100%)
rambeau | Oct 03, 13:05 |  3:17:28 | world_yellowdog (93.71%)
```
per the configuration:
```
Customization:
  UseServerTime: true
  Formatting:
    DecimalPoints: '#.##'
    TimeAmount:
      Year: '1 year, '
      Years: '%years% years, '
      Month: '1 month, '
      Months: '%months% months, '
      Day: '1 day '
      Days: '%days% days '
      Hours: '%hours%:'
      Minutes: '%zero%%minutes%:'
      Seconds: '%zero%%seconds%'
```

**Comment by Rsl1122** [comment]
Weird that's what it's supposed to be doing. Probably a frigging bug again

**Comment by SlimeDog** [comment]
Hmmm. Now it's working. Probably I forgot to **/plan reload**

Mea culpa (they aren't all bugs; sometimes it's operator error).

**Comment by SlimeDog** [comment]
Ooops! There is the 0 edge case. See playtime for Drew_Scorpion.
```

Player        | Active   | Playtime      | Sessions | Registered         | Last seen     | Geolocation
------------- | -------- | ------------- | -------- | ------------------ | ------------- | --
rambeau       | Active   | 1 day 1:16:49 | 15       | Apr 22 2015, 22:49 | Oct 04, 12:49 | Not Known
Drew_Scorpion | Inactive | %zero%0       | 0        | Feb 03 2017, 09:47 | -             | Not Known
```

**Comment by SlimeDog** [comment]
Verified fix in 4.0.2-DEV1
